### PR TITLE
Elegant monolith

### PR DESCRIPTION
**Pull request by georgolden**
This PR is for review only

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import serverConf from './config/server.js';
+import { initServices } from './init/initServices.js';
+import { initRoutes } from './init/initRoutes.js';
+
+const server = fastify({ logger: true });
+
+const { bus } = initServices();
+const fastifyRoutes = initRoutes({ bus });
+
+console.dir(fastifyRoutes, { depth: 10 });
```
Comment (d4rl1nn) [inline src/app.js:11]: remove console.dir

File: src/fastify/mapper.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export class FastifyRouteMapper {
+  constructor({ bus }) {
+    this.bus = bus;
+  }
+
+  toFastifyRoute(name, route) {
+    console.dir({ route });
```
Comment (d4rl1nn) [inline src/fastify/mapper.js:7]: remove console.dir

**Review by d4rl1nn — APPROVED** [review]
Perfect code :+1:

**Comment by georgolden** [comment]
This will never be used as a main template. Imperative & Functional hybrid is tinier more testable and more readable

**Pull request closed without merging**